lib/matplotlib/tests/test_backend_qt.py

```diff
@@ -114,6 +114,42 @@ def test_fig_close(backend):
     assert init_figs == Gcf.figs
 
 
+@pytest.mark.backend('Qt5Agg')
+def test_fig_close_ioff(qt_module):
+    # Turn off interactive mode
+    plt.ioff()
```

Comment:
do you need this line for the tests to pass?

Reply:
I think so, yes. Otherwise plt.show() isn't a blocking call, which I think means that an entirely different branch of code is executed. The test would pass, but it wouldn't be testing the fix I made. But I'm working off of memory, I can double-check later.

Reply:
Not sure this really matters, but OTOH it probably doesn't hurt, so heh.

Reply:
Oh that's true, it might be a no-op...